docs/docker-machine-drivers.md

```diff
@@ -74,7 +84,7 @@ https://github.com/zchee/docker-machine-driver-xhyve
 From: https://github.com/Homebrew/homebrew-core/blob/master/Formula/docker-machine-driver-xhyve.rb
 ```
 
-To install the latest driver version via brew:
+To install the driver with the latest version via brew:
```

Comment:
"with the latest version"!? I think in this case it would be "in its latest version", but tbh I don't see what's wrong with the initial wording.